modules/040-control-plane-manager/docs/FAQ.md

```diff
@@ -35,6 +35,15 @@ Adding a master node to a static or hybrid cluster has no difference from adding
      --ssh-host <MASTER-NODE-0-HOST>
    ```
 
+1. **In the installer container**, run the following command to check the state before working:
+
+   ```bash
+   dhctl terraform check --ssh-agent-private-keys=/tmp/.ssh/<SSH_KEY_FILENAME> --ssh-user=<USERNAME> \
+     --ssh-host <MASTER-NODE-0-HOST> --ssh-host <MASTER-NODE-1-HOST> --ssh-host <MASTER-NODE-2-HOST>
+   ```
+
+   The response should tell you that Terraform does not want to change anything.
+
```

Comment:
We do this check after master replicas change, cluster needs to be converged at this point.